modules/osdk-cli-reference-olm-status.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-olm-status_{context}"]
+= status
+
+`olm status` subcommand gets the status of the PLM installation in your cluster.
```

Comment:
Question: Do you know if `PLM` is typo for `OLM`?  If it is not a typo, please consider expanding that abbreviation.

Reply:
It's a typo.